test/cli/verify.py

```diff
@@ -42,5 +43,5 @@ def verify(base_url, keyid, digest, sig):
 
     base_url, keyid, digest, sig = get_args()
 
-    verify(base_url, keyid, digest, sig)
+    verify(base_url, keyid, digest, sigwaitinfo)
```

Comment:
the sig used in line 20, 24, 44, if not used anymore, delete it.

Reply:
it will be changed